### PR TITLE
allow more flexibility when using database urls and postgresql

### PR DESCRIPTION
**Pull request by rkingon**

### CODE REVIEW

File: src/config/DbConfig.php [hunk 1]
```diff
@@ -121,7 +121,12 @@ public function init()
         if ($this->url !== null) {
             $url = parse_url($this->url);
             if (isset($url['scheme'])) {
-                $this->driver = $url['scheme'];
+                $scheme = strtolower($url['scheme']);
+                if ($scheme === "pgsql" || strpos($scheme, "postgres") !== false) {
+                    $this->driver = self::DRIVER_PGSQL;
+                } else {
```
Comment (rkingon) [inline src/config/DbConfig.php]: potentially better off with a `else if ($scheme === "mysql")`

**Comment by brandonkelly** [comment]
What’s an example DB connection URL that requires the added flexibility?

**Comment by rkingon** [comment]
heroku gives the following:
`postgres://user:password@host/database`

**Comment by brandonkelly** [comment]
👍 Thanks

**Comment by rkingon** [comment]
the standard for postgres url's is actually `postgresql://user:password@host/database` so i used strpos so it handles `postgres|postgresql`

**Pull request merged**